R/accounts.R

```diff
@@ -73,21 +73,6 @@ connectApiUser <- function(account = NULL, server = NULL, apiKey = NULL, quiet =
     stop("You must specify a server to connect to.")
   }
 
-  # if account is specified and we already know about the account, get the User
-  # ID so we can prefill authentication fields
-  userId <- 0
-  userAccounts <- accounts(target$name)
-  if (!is.null(account) && !is.null(userAccounts)) {
-    if (account %in% userAccounts[,"name"]) {
-      accountDetails <- accountInfo(account, target$name)
-      userId <- accountDetails$accountId
-      if (!quiet) {
-        message("The account '",  account, "' is already registered; ",
```

Comment:
I think we need to keep this message so that you know if you're overwriting an existing account.

Reply:
👍 I'll add back the two sections.